src/components/LocationPage/ChartBlock.tsx

```diff
@@ -51,6 +51,9 @@ function ChartBlock(props: {
   const futureProjectionsDisabled =
     FUTURE_PROJECTIONS_DISABLED_STATES.includes(props.stateId) &&
     !props.countyId;
+  const haveFutureProjections =
```

Comment:
should this be `hasFutureProjections`? hah what's the subject here ✏️ ?

Reply:
Heh.  In my mind the subject is "we" ("do we have future projections? yes/no").